Tasks/a3_check_brackets.py

```diff
@@ -4,5 +4,20 @@ def check_brackets(brackets_row: str) -> bool:
 	:param brackets_row: input string to be checked
 	:return: True if valid, False otherwise
 	"""
+	brackets_stack = []
+	brackets_dict = {')': '(', '}': '{', ']': '['}
+	for i in brackets_row:
+		if i not in [')', '(', '}', '{', ']', '[']:
```

Comment:
```suggestion
		if i not in {')', '(', '}', '{', ']', '['}:
```
Поиск будет за O(1), а не за O(N)

Reply:
Неблокирующий